src/njson.erl

```diff
@@ -17,26 +17,35 @@
 -export([decode/1, encode/1, encode/2]).
 
 %%% TYPES
--type t() :: undefined | boolean() | number() | binary() | [t()] | #{binary() => t()}.
+-type t() :: null | boolean() | number() | binary() | [t()] | #{binary() => t()}.
```

Comment:
I would say `undefined` is still part of a valid representation of a JSON. Its counterpart is the empty binary, right?